README.md

```diff
@@ -39,6 +39,7 @@
 - 💵 [Affinity Photo](https://affinity.serif.com/en-gb/photo) *(Windows, Mac, iOS)*
 - 💵 [Photoline](https://www.pl32.com) *(Windows, Mac)*
 - 💵 [Acorn](https://flyingmeat.com/acorn) *(Mac)*
+- ✨ [digiKam] (https://www.digikam.org/) *(Windows, Mac, Linux)*
```

Comment:
Free and open-source are always on top of lists according to the paragraph at head of the README